src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -115,7 +136,7 @@
                 escape = false;
                 switch (c) {
                     case ('<') : if (index == 0) break; // if we see a open bracket at the beginning, ignore it
-                    case ('>') : if (index == valueLine.length()-1) ret.put(key,builder.toString().trim()); break; // if we see a close bracket, and we're at the end, add an entry to our list
+                    case ('>') : if (index == valueLine.length()-1 && !key.isEmpty()) ret.put(key,builder.toString().trim()); break; // if we see a close bracket, and we're at the end, add an entry to our list
```

Comment:
All the cases in this `switch` should be reformatted to follow the code conventions
```
case '>':
    if (index == valueLine.length() - 1 && !key.isEmpty()) {
        // if we see a close bracket, and we're at the end, add an entry to our list
        ret.put(key, builder.toString().trim());
        break;
    }
```

Reply:
Mixing reformats with logic changes in a single PR is a bad practice IMHO. If you want to do a "pure reformat" PR, we can make a separate one.